apps/web/playwright/reschedule.e2e.ts

```diff
@@ -210,4 +211,187 @@ test.describe("Reschedule Tests", async () => {
     expect(newBooking).not.toBeNull();
     expect(newBooking?.status).toBe(BookingStatus.ACCEPTED);
   });
+
+  test.describe("Reschedule for booking with seats", () => {
+    test("Should reschedule booking with seats", async ({ page, users, bookings }) => {
```

Comment:
Let's add to the tests:
- If rescheduled/cancelled, make sure the seat becomes available again.